test/simple-browser-extension/src/interfaces/IAccountCallSCRequest.ts

```diff
@@ -0,0 +1,14 @@
+import { Args } from "@massalabs/massa-web3";
+
+/**
+ * This interface represents the request object to send to the content script to 
+ * interact with a deployed smart contract.
+ */
+export interface IAccountCallSCRequest {
+    nickname: string, // account nickname
+    name: string, // function name
+    at: string, // contract address
+    args: Args, // function arguments
+    coins: number, // coins to send to the block creator
```

Comment:
coins are normaly expressed in bigint, no ?

Reply:
Here the interface is used for massastation but also the browser extension. So the code needs to be compatible for both.
This being said, are you sure a u64 can be expressed using JSON ?

Reply:
My bad :
"the JSON standard does not have a built-in numeric type that corresponds to u64 (unsigned 64-bit integer). However, you can still include a u64 value in a JSON object without converting it to a string"

Reply:
Now I have a "Error: Axios error: TypeError: Do not know how to serialize a BigInt" while calling callSC

Reply:
You are right but axios won't allow it natively. I'll try to make it work